Makefile

```diff
@@ -408,6 +408,8 @@ compiler_brand := $(shell $(CC) --version | grep -o -m1 clang || echo gcc)
 cc_version := $(shell $(CC) -dumpversion)
 
 WARN_FLAGS = 
+WARN_FLAGS += -Wno-unused-value
+WARN_FLAGS += -Wno-unused-variable
```

Comment:
I usually appreciate having these warnings, as it indicates places where issues could arise. Could we go through and fix those issues? Or are they specific to third-party libraries?

Reply:
Removed warning suppression. I'll just make a shell script to set WARN_FLAGS env on my end locally.

As for the warnings, they are primarily originating from 3rd party headers. Vorbis is the main offender. A couple of Lua and assertion macros are also sources. The warnings reported between gcc and clang actually differ wildly too, as well as across different versions of clang. It seems very inconsistent which compilers - and which versions of these compilers - judge unused values or variables as worth reporting. This is evidence that compilers are in some cases already trying to reduce the volume of warnings these tend to generate, and have yet to arrive at any consensus on what's good or right.